models/setup.go

```diff
@@ -0,0 +1,18 @@
+package models
+
+import (
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/sqlite"
+)
+
+var DB *gorm.DB
+
+func ConnectDatabase() {
+	database, err := gorm.Open("sqlite3", "test.db")
+
+	if err != nil {
+		panic("Failed to connect to database")
+	}
+	database.AutoMigrate(&User{})
+	DB = database
```

Comment:
this file should be inside `db` package. A package or folder named `db` in root folder

Reply:
change done